cmd/pbm/main.go

```diff
@@ -20,11 +20,12 @@ var (
 	pbmCmd = kingpin.New("pbm", "Percona Backup for MongoDB")
 	mURL   = pbmCmd.Flag("mongodb-uri", "MongoDB connection string").String()
 
-	configCmd     = pbmCmd.Command("config", "Set, change or list the config")
-	configListF   = configCmd.Flag("list", "List current settings").Bool()
-	configFileF   = configCmd.Flag("file", "Upload config from YAML file").String()
-	configSetF    = configCmd.Flag("set", "Set the option value <key.name=value>").StringMap()
-	configShowKey = configCmd.Arg("key", "Show the value of a specified key").String()
+	configCmd           = pbmCmd.Command("config", "Set, change or list the config")
+	configRsyncBcpListF = configCmd.Flag("resync", "Resync backup list with the current store").Bool()
```

Comment:
Just an idea (and not a blocker): maybe this should be "force-resync" so users don't get the idea that it is something they're obliged to do in normal circumstances.